schemas/user_agent.yml

```diff
@@ -15,45 +13,41 @@
     - name: device
       type: keyword
       description: >
-        The name of the physical device.
+        Name of the physical device.
     - name: version
       type: keyword
       description: >
         Version of the physical device.
     - name: major
       type: long
       description: >
-        The major version of the user agent.
+        Major version of the user agent.
     - name: minor
       type: long
       description: >
-        The minor version of the user agent.
+        Minor version of the user agent.
     - name: patch
       type: keyword
       description: >
-        The patch version of the user agent.
+        Patch version of the user agent.
     - name: name
       type: keyword
       example: Chrome
       description: >
-        The name of the user agent.
+        Name of the user agent.
     - name: os.name
       type: keyword
       description: >
-        The name of the operating system.
+        Name of the operating system.
     - name: os.version
       type: keyword
       description: >
         Version of the operating system.
     - name: os.major
       type: long
       description: >
-        The major version of the operating system.
+        Major version of the operating system.
     - name: os.minor
       type: long
       description: >
-        The minor version of the operating system.
-    - name: os.name
```

Comment:
Nice find. I think I should add a check to the script that throws an error if we have a field twice.